spec/services/doc_auth/lexis_nexis/responses/true_id_response_spec.rb

```diff
@@ -151,10 +152,34 @@
   end
 
   context 'when response is not a success' do
-    it 'it produces appropriate errors without liveness' do
+    it 'produces appropriate errors without liveness' do
       output = described_class.new(failure_response_no_liveness, false, config).to_h
       errors = output[:errors]
-
+      expect(output.to_h[:log_alert_results]).to eq(
+        passed:
+          { '2d_barcode_read': { no_side: 'Passed' },
```

Comment:
In my ideal world, we would not break these out into`passed` or `failed` keys like this.

This structure means if I want to query the % of `2d_barcode_read` that pass in Cloudwatch, I have to `coalesce()` two fields:
- `passed.2d_barcode_read.no_side`: `Passed`
- `failed.2d_barcode_read.no_side`: `Caution`

My hope is that we could have one key for both pass and fail, which simplifies querying, ex
- `2d_barcode_read` [values: `Passed` | `Caution` | `Attention` | `Failed`, etc ]

Reply:
I'm having a little trouble squaring this comment with the previous one: https://github.com/18F/identity-idp/pull/6612#discussion_r926888904

The reason for including the `failed` and `passed` keys was to make it easy to detect an error, such as `2d_barcode_read` by doing something like `ispresent(failed.2d_barcode_read)` in cloudwatch. That got kind of munged because we had the suggestion of including side information which does make it more difficult to coalesce errors.

That pattern suggested here that gives us one key for pass and fail removes the side information which it sounded like you were suggesting we include. Is there a structure you have in mind that we could log that includes side information?

Reply:
Parissa and I chatted on Slack yesterday, the changes in d785166622455064c064b6a1b0aa2770fab2f6ad get us to what I was trying to request

I think my comment before was clear in my head but not clear to others. When I said
> can we have a combined alert_results that includes logging both passed and failed alerts? It helps us look at the % of time each signal passes
I was trying to say "let's not have separate hashes/keys for passed and failed, let's have one big one" so the goal is `2d_barcode == "Failed"` is now our CW check for failed instead of having to do `ispresent()` to detect an error

The current structure allows for things like ` | stats count(*) by 2d_barcode_read` to see the breakdown of statuses for that one check

Reply:
Yes so before we were looping through just the failed ones `alerts[:failed].each_with_object({})` now it loops through the failed and success hashes.